tests/test_parser.py

```diff
@@ -7,21 +7,24 @@
 from gdtoolkit.parser import parse
 
 
-OK_DATA_DIR = "valid-gd-scripts"
-NOK_DATA_DIR = "invalid-gd-scripts"
+OK_DATA_DIR = "./valid-gd-scripts"
+NOK_DATA_DIR = "./invalid-gd-scripts"
 GODOT_SERVER = "godot-server"
 
 
 def pytest_generate_tests(metafunc):
+    this_directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This removes the need for changing directory in a tox pyenv dedicated to pytest only, allowing us to work with a single environment.